project/src/test/java/com/google/sps/UserProfileEventsTest.java

```diff
@@ -179,4 +177,39 @@ public void verifyGetUserEvents_notLoggedIn_redirectToLogin() throws Exception {
     Mockito.verify(response).sendRedirect("/index.html");
   }
 
+  /*Test to get all events logged in user is participating in */
+  @Test
+  public void getUserEventsParticipating() throws IOException {
+    //Logged in user
+    User user = TestUtils.newUserWithEmail(EMAIL);
+    String userEmail = user.getEmail(); 
+    SpannerTasks.insertOrUpdateUser(user);
+    authenticationHelper
```

Comment:
Put this in a helper function to abstract away authentication details?

Reply:
lmk if you'd rather I put that in my eventsHosting PR, we'd be using the same one